.circleci/config.yml

```diff
@@ -0,0 +1,265 @@
+            - run:
+                name: dependencies
+                command: ./.circleci/set-up-globals.sh
+
+            - run:
+                name: run unit tests
+                command: ./.circleci/unit-test.sh
+
+            - save_cache:
+                key: composer-cache
+                paths:
+                    - $HOME/.composer/cache
+            
+            - save_cache:
+                key: terminus-install
+                paths:
+                    - $HOME/terminus
+                    - $HOME/.terminus
+
+    deploy_to_pantheon:
+        <<: *defaults
+        steps:
+            - checkout
+
+            - restore_cache:
+                keys:
+                    - composer-cache
+                    - terminus-install
+
+            - attach_workspace:
+                # Must be absolute path or relative path from working_directory
+                at: /tmp
+
+            - run:
+                name: copy vendor
+                command: cp -r /tmp/vendor .
+            
+            - run:
+                name: copy web
+                command: cp -r /tmp/web .
+
+            - run:
+                name: dependencies
+                command: ./.circleci/set-up-globals.sh
+
+            - run:
+                name: deploy to Pantheon
+                command: ./.circleci/deploy-to-pantheon.sh
+
+            - save_cache:
+                key: composer-cache
+                paths:
+                    - $HOME/.composer/cache
+            
+            - save_cache:
+                key: terminus-install
+                paths:
+                    - $HOME/terminus
+                    - $HOME/.terminus
+
+    behat_test:
+        <<: *defaults
+        steps:
+            - checkout
+
+            - restore_cache:
+                keys:
+                    - composer-cache
+                    - terminus-install
+
+            - attach_workspace:
+                # Must be absolute path or relative path from working_directory
+                at: /tmp
+
+            - run:
+                name: copy vendor
+                command: cp -r /tmp/vendor .
+            
+            - run:
+                name: copy web
+                command: cp -r /tmp/web .
+            
+            - run:
+                name: dependencies
+                command: ./.circleci/set-up-globals.sh
+
+            - run:
+                name: run Behat tests
+                command: ./.circleci/behat-test.sh
+
+            - save_cache:
+                key: composer-cache
+                paths:
+                    - $HOME/.composer/cache
+            
+            - save_cache:
+                key: terminus-install
+                paths:
+                    - $HOME/terminus
+                    - $HOME/.terminus
+
+workflows:
+  version: 2
+  build_test:
+    jobs:
+      # Build the site first
+      - build
+      # Run code sniffing and unit testing
+      - code_sniff:
+          requires:
+            - build
```

Comment:
phpcs is installed as a de dependency with Composer. It was easier to just keep the build step, which installs all dependencies, so the `phpcs` bin is available.

This does commit dev dependencies to Pantheon though, which we should probably avoid.